src/frontend/package.json

```diff
@@ -6,20 +6,29 @@
     "develop": "next dev -p 8000",
     "build": "next build",
     "start": "next start",
+    "test": "jest --watchAll",
```

Comment:
We don't want to use watch when running `test`, it needs to be a separate command so that CI doesn't hang forever.